ts/src/gameClasses/EntityTrack.ts

```diff
@@ -0,0 +1,155 @@
+class EntityTrack {
```

Comment:
still not happy with the name of this class. it might be tracking entities, but it does a lot more than that, think `EntityManager` fits better. `ige` property could me called `entityManager` or even more elegant `entities`:)

Reply:
I've been brainstorming this and my main thought is that it can't be a blanket reference to entities because this is only 'entities we are rendering'